### PR TITLE
Extend AppCompatActivity instead of FragmentActivity

### PR DESCRIPTION
**Pull request by cr5315**
Also updated support libs and added my app to the readme

### CODE REVIEW

**Comment by PaoloRotolo** [comment]
Can you upload the pull again please? It seems that GitHub can't merge it automatically.

**Comment by cr5315** [comment]
It's because README.md changed from the version I have in my fork. I'll see what I can do about that.

**Comment by cr5315** [comment]
All good :smile:

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java [hunk 1]
```diff
@@ -5,19 +5,23 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentActivity;
 import android.support.v4.view.ViewPager;
+<<<<<<< HEAD
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java]: @cr5315 this should be removed.

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java [hunk 2]
```diff
@@ -5,19 +5,23 @@
...
+import android.support.v7.app.AppCompatActivity;
+import android.view.*;
+=======
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java]: @cr5315 this line should be removed.

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java [hunk 3]
```diff
@@ -5,19 +5,23 @@
...
 import android.view.KeyEvent;
 import android.view.View;
 import android.view.Window;
 import android.view.WindowManager;
+>>>>>>> upstream/master
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro2.java]: @cr5315 this line should be removed.

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java [hunk 4]
```diff
@@ -6,12 +6,16 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentActivity;
 import android.support.v4.view.ViewPager;
+<<<<<<< HEAD
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: @cr5315 this line should be removed.

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java [hunk 5]
```diff
@@ -6,12 +6,16 @@
...
+import android.support.v7.app.AppCompatActivity;
+import android.view.*;
+=======
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: @cr5315 this line should be removed.

File: library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java [hunk 6]
```diff
@@ -6,12 +6,16 @@
...
 import android.view.KeyEvent;
 import android.view.View;
 import android.view.Window;
 import android.view.WindowManager;
+>>>>>>> upstream/master
```
Comment (raphaelcuba) [inline library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java]: @cr5315 this line should be removed.

**Comment by raphaelcuba** [comment]
Great pr @cr5315, but has some fixes to resolve.

**Comment by cr5315** [comment]
Sorry about that, everything should be good to go now. I was able to build and run the sample app after making the changes and everything worked.

**Comment by raphaelcuba** [comment]
lgtm, thanks @cr5315! o/

**Comment by PaoloRotolo** [comment]
Thanks @cr5315 and @raphaelcuba. Merging :)

**Pull request merged**